configure.ac

```diff
@@ -594,7 +594,7 @@ AS_IF([test "x${RKT_RUN_FUNCTIONAL_TESTS}" = 'xyes' -o "x${RKT_RUN_FUNCTIONAL_TE
 # Check if arch is supported for coreos/kvm flavor.
 AS_CASE([${RKT_ACI_ARCH}],
         dnl supported arches
-        [aarch64 | amd64], [],
+        [aarch64 | amd64 | ppc64le], [],
```

Comment:
On a second look, this shouldn't be here as you are using stage1-host and we don't have any coreos-distro support for ppc64. Can you please just remove this change?